data/bash-completion/fwupdtool

```diff
@@ -11,6 +11,7 @@ _fwupdtool_cmd_list=(
 	'enable-test-devices'
 	'emulation-tag'
 	'emulation-untag'
+	'emulation-load'
```

Comment:
you should also add this to the `	get-details|install|install-blob|firmware-dump|firmware-read)` case.

Reply:
Good idea; I added a new block as we don't want `_show_device_ids` for this.